var/spack/repos/builtin/packages/py-heat/package.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyHeat(PythonPackage):
+    """Heat is a flexible and seamless open-source software for high performance data analytics
+    and machine learning. It provides highly optimized algorithms and data structures for tensor
+    computations using CPUs, GPUs and distributed cluster systems on top of MPI."""
+
+    homepage = "https://github.com/helmholtz-analytics/heat/"
+    url = "https://github.com/helmholtz-analytics/heat/archive/refs/tags/v1.3.0.tar.gz"
+
+    maintainers("mrfh92", "ClaudiaComito", "JuanPedroGHM")
+
+    version("1.3.0", sha256="2e66e82df87431075be633bc921a4953dd258dfa91ed31283c964bcb3dea1ffc")
+
+    depends_on("py-mpi4py@3:", type=("build", "run"))
+    depends_on("py-numpy@1.2:", type=("build", "run"))
+    depends_on("py-torch@1.8:2.0.1", type=("build", "run"))
+    depends_on("py-scipy@0.14:", type=("build", "run"))
+    depends_on("py-pillow@6:", type=("build", "run"))
```

Comment:
There is a virtual package you can use here
```suggestion
    depends_on("pil@6:", type=("build", "run"))
```